### DISCUSSION TITLE
Icon Request: icon-trash-open/icon-trash-open-o

### DISCUSSION THREAD

**Discussion by maximkott** [FortAwesome/Font-Awesome#21153] [Icon Requests]
A counterpart to the "icon-trash" would be great. Same icon, but the bucket should be open.

**Comment by ondrakoupil** [comment]
+1, I'd appreciate that too.

**Comment by alopatin** [comment]
+1 Would let you do neat things with hover state.

**Comment by moimael** [comment]
+1 I'd really find this useful !

**Comment by Fededark** [comment]
+1 It would be nice!

**Comment by anand9** [comment]
+1 will have a cool effect with trash-can hover :)

**Comment by StephaneJuban** [comment]
+1, it would be great for a trash hover effect

**Comment by GiboMac** [comment]
+1 Yes Please! :D

**Comment by aaronsummers** [comment]
+1 from me too, could do with this right now

**Comment by queengooborg** [comment]
I'm surprised this isn't an icon yet.  +1, would love to see this in FontAwesome!

**Comment by pralhadstha** [comment]
Is this icon in WIP??? +1. When  will this be introduced?

**Comment by AlbertMN** [comment]
+1

**Comment by seustachi** [comment]
+1

**Comment by Ppito** [comment]
+1

**Comment by mannyp2** [comment]
+1

**Comment by hpvd** [comment]
+1

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
